INSTALL.md

```diff
@@ -8,17 +8,19 @@ INSTALL file for Faiss (Fair AI Similarity Search)
 Install via Conda
 -----------------
 
-The easiest way to install FAISS is from anaconda. We regularly push stable releases to conda channel. FAISS conda package depends on mkl and numpy package shipped in conda.
+The easiest way to install FAISS is from anaconda. We regularly push stable releases to conda channel. FAISS conda package is built with conda gcc, depends on libgcc, mkl and numpy package shipped in conda in runtime.
 
 Currently we support faiss-cpu on both Linux and OSX platforms. We also provide faiss-gpu compiled with CUDA8.0/CUDA9.0/CUDA9.1 on Linux systems.
 
 You can easily install it by
 
 ```
+# Install conda dependencies
+conda install numpy mkl libgcc
```

Comment:
This step shouldn't be necessary, as those are listed as conda dependencies, right?

Reply:
Yep they are. It's basically only needed in the rare condition: when install the tar.gz directly. I could remove them